packages/docs/docs/components/code-block.mdx

```diff
@@ -97,10 +96,34 @@ Code may be "selected" to call attention to it. The default behavior is to
 desaturate any unselected text.
 
 A selection can be made using the `selection` property when the `CodeBlock` is
-first created. Selections are specified using two `[line, position]` arrays,
-where `line` and `position` are zero indexed, and `position` is an insert
-position between characters. So `[[0, 5], [1, 2]]` will select the sixth
-character on the first line through to the second character on the second line.
+first created. Selections are specified using two `[line, position]` arrays
```

Comment:
This section should come after we explain the helpers, since 90% of the time the helpers make more sense than the nested arrays.